in_toto/gpg/common.py

```diff
@@ -730,7 +728,13 @@ def parse_signature_packet(data, supported_signature_types=None,
     if subpacket_type == SIG_CREATION_SUBPACKET:
       info["creation_time"] = struct.unpack(">I", subpacket_data)[0]
 
-    info["subpackets"][subpacket_type] = subpacket_data
+    info["subpackets"][subpacket_type] = (subpacket_data, is_hashed)
+
+  # Warn if expiration subpacket is not hashed
+  if 9 in (info["subpackets"]):
```

Comment:
instead of 9, could you use the constant? (I think it is KEY_EXPIRATION_SUBPACKET).